libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -350,12 +356,8 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
 
     /** @hidden Subscribe to carousel service events */
     private _subscribeServiceEvents(): void {
-        this._carouselService.activeChange
-            .pipe(takeUntil(this._onDestroy$))
-            .subscribe((event) => this._onSlideSwipe(event));
-        this._carouselService.dragStateChange
-            .pipe(takeUntil(this._onDestroy$))
-            .subscribe((event) => this._onSlideDrag(event));
+        this._carouselService.activeChange.subscribe((event) => this._onSlideSwipe(event));
```

Comment:
How's this subscription being destroyed?

Reply:
here https://github.com/SAP/fundamental-ngx/pull/7529/files/254b75e477b072ba95c8a9c34483892e36d44c39#diff-6a760f57d72b55490dd32c11c868561f957d368ee7ff27b7401b9ffb83e633f6R99